packages/vike-react-styled-components/config.ts

```diff
@@ -0,0 +1,34 @@
+export { config as default }
+
+import type { IStyleSheetManager, ServerStyleSheet } from 'styled-components'
+import type { Config } from 'vike/types'
+
+const config = {
+  name: 'vike-react-styled-components',
+  require: {
+    vike: '>=0.4.203',
+    'vike-react': '>=0.4.13',
+  },
+  onAfterRenderHtml: 'import:vike-react-styled-components/__internal/onAfterRenderHtml:onAfterRenderHtml',
+  onBeforeRenderHtml: 'import:vike-react-styled-components/__internal/onBeforeRenderHtml:onBeforeRenderHtml',
+  Wrapper: 'import:vike-react-styled-components/__internal/Wrapper:Wrapper',
+  meta: {
+    styleSheetManager: {
```

Comment:
How about `+style-components` instead? Feel free to disagree but I kinda like this convention.

Reply:
How about this:

```js
// +styledComponents.js

export default {
   styleSheetManager
}
```

Best of both worlds if you ask me.

>  but I'm not sure if we can use a hyphen (-).

Good point. ~~I don't have any opinion on this (yet?). We can go for `+styledComponents.js`.~~ Actually, it's better to go for `+styledComponents.js` to keep the export name consistent.